docs/advanced/debug.md

```diff
@@ -3,67 +3,88 @@ title: Enable Kubectl logs/exec to debug pods on the edge
 sidebar_position: 3
 ---
 
-## Prepare certs
```

Comment:
the main motivation is to keep the all information in here about `how to enable kubectl log/exec`, and use the ref links from elsewhere for the maintenance cost and avoiding the complication.